spec/fixtures/habitat/plan.sh

```diff
@@ -0,0 +1,10 @@
+pkg_name=fixture
+pkg_origin=chef
+pkg_version="0.1.0"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=("Apache-2.0")
+pkg_deps=(
+  core/glibc
```

Comment:
You should put 2 or 3 licenses in here 😉

Reply:
I suppose I could. I know it works since I've run it on other more robust projects, but I will update the tests.